packages/ragbits-cli/src/ragbits/cli/__init__.py

```diff
@@ -2,11 +2,25 @@
 import pkgutil
 from pathlib import Path
 
-from typer import Typer
+import typer
 
 import ragbits
 
-app = Typer(no_args_is_help=True)
+from .app import CLI
+
+app = CLI(no_args_is_help=True)
+
+
+@app.callback()
+def output_type(
+    output: str = typer.Option("text", "--output", "-o", help="Set the output type (text or json)"),
```

Comment:
This should also be typed with the `OutputType` enum. See here: https://typer.tiangolo.com/tutorial/parameter-types/enum/

Otherwise the command will crash (generate a stack trace) for wrong values from user, instead of generating a user friendly message. Using an enum will also automatically document the possible values.

Reply:
done - however it was not possible to make a typing exactly as in documentation.